### DISCUSSION TITLE
Wunsch: Überdenken/Verbessern Heizungsgeräte

### DISCUSSION THREAD

**Discussion by emufan** [SukramJ/aiohomematic#465] [Ideas]
Hier habe ich mehrere Themen als Wunsch, bzw. Verbesserungsvorschläge. Teile davon gab es "unten" schon mal als Frage und wurde andiskutiert, aber vielleicht kann ich das nochmal darstellen, was ich meine.

### Thermostate

Vorher (hier immer als Core-(Integration):

![image](https://user-images.githubusercontent.com/22775515/170858629-d9c18978-4757-46b8-ac04-a766536c33ca.png)

Jetzt:

![image](https://user-images.githubusercontent.com/22775515/170858640-fbdf0ddb-565b-4fa3-9c90-25248fb86b47.png)

- Muss das Profil wirklich in den Status oben rein? Ist a. redundant, da unten sowieso und b. reduziert die Anzeigebreite ungemein und verkürzt die nicht redundanten Infos. Name wird abgekürzt und dadurch selbst die Zeit.

Und so in Entity-Card:

![image](https://user-images.githubusercontent.com/22775515/170858673-7387f030-aa6b-463a-ab5b-755283a356a5.png)

- Hier kann man so gar nichts mehr sehen. Wenn man dort das Profil haben überhaupt möchte, könnte man es sich ja auch selber als Second Info setzten, o.Ä.

### Ventile

Vorher:

![image](https://user-images.githubusercontent.com/22775515/170859051-8b94bf4f-c015-4df7-8dcf-201f3303db34.png)

Jetzt:

![image](https://user-images.githubusercontent.com/22775515/170859072-afcf5e7a-d267-4828-b121-8ec89539d90e.png)
![image](https://user-images.githubusercontent.com/22775515/170859126-c2b66ce1-e264-476c-bdb4-9224219a3046.png)

- Ja. Tatsächlich so. Noch extremer als bei den Thermostaten. Mit Scrollbar. Sonst eigentlich noch nirgendwo gesehen in HA. Und Namen und Timestamp so wie oben.

### Heizungsgruppen

Vorher:

![image](https://user-images.githubusercontent.com/22775515/170858476-a06b6784-738f-44ca-8175-e5d6187edf73.png)

Jetzt:

![image](https://user-images.githubusercontent.com/22775515/170858506-3661fe1b-b271-458f-9997-3d9046141516.png)

- Quasi wie Thermostate, s.o.

### Ventilstatus (in Gruppen)

Neu: 

![image](https://user-images.githubusercontent.com/22775515/170859152-e2eb7d28-75ef-4a11-9a9a-dddca2d8c909.png)

- Super. Aber leider nur bei den Ventilen. Nur bedient man bei HM ja die Gruppen und nicht das Ventil. Und dann ist es auch nicht im Verlauf zu sehen, wenn die User dann die Gruppen ins UI nehmen (müssen).

![image](https://user-images.githubusercontent.com/22775515/170859431-ddc88a19-6d11-4c4c-a572-718f0acdb8a3.png)

- Ich weiß, dass das nur eine virtuelle Zusammenfassung im HM-UI ist, aber dennoch sieht man auch dort den Stand bei Gruppen

![image](https://user-images.githubusercontent.com/22775515/170859299-9d6294b8-52c9-4ac5-91b2-5cd1f9282e87.png)

- Könntest Du das nicht auch noch zusätzlich "hochmappen/hochkopieren" aus den Ventilen in die Gruppe, sodass man das dann auch dort hat? Der sonstige Verlauf ist ja auch vorhanden, d.h. Soll und ist, etc.

**Comment by SukramJ** [comment]
Thermostate:
Die UI kann ich nicht beeinflussen. Hab ich aber [hier](https://github.com/danielperna84/hahomematic/discussions/314#discussioncomment-2256726) schon erklärt.

**Comment by SukramJ** [comment]
Ventilstatus:
Kann ich machen. 
Dann sieht die Gruppe in der UI aus wie dein Screenshot unter Ventile/jetzt.

**Comment by emufan** [comment]
Das wäre ja fantastisch!

**Comment by emufan** [comment]
Funktioniert. Vielen Dank.

![image](https://user-images.githubusercontent.com/22775515/170943427-9aea6018-93d9-4a99-89bd-5d97202d48e1.png)

Das mit den Bezeichnungen finde ich weiter unglücklich. Musst Du denn alles übergeben, bzw. warum war das bei der alten Integration anders? Einfach nur unvollständiger oder wie kommt es zu den Unterschieden?

**Reply by SukramJ** [reply]
Hab ich alles bereits in der verlinkten Diskussion geschrieben.
-> die alte Integration ist unvollständiger.
